src/routes/group/chat/[id]/+page.svelte

```diff
@@ -62,6 +66,38 @@
 	}
 
 	$: chat = $chats.chats.get($page.params.id)
+
+	$: inviter = chat?.users.find((user) => user.address === chat?.inviter)
+
+	function join() {
+		chats.update((state) => {
```

Comment:
I added some helper functions that can be potentially used here to simplify this: https://github.com/logos-innovation-lab/waku-objects-playground/pull/306/

Maybe it can be done in another PR when the decline will be activated.

Reply:
Awesome, thanks!